.pre-commit-config.yaml

```diff
@@ -0,0 +1,56 @@
+# pre-commit is a tool that you run locally
+# to perform a predefined set of tasks manually and/or
+# automatically before git commits are made. 
+# Here we are using pre-commit with the precommit.ci bot to implement 
+# code fixes automagically in pr's. You will still want to install pre-commit 
+# to run locally
+# Config reference: https://pre-commit.com/#pre-commit-configyaml---top-level
+# To run on a pr, use: pre-commit.ci run
+# Common tasks
+#
+# - Run on all files:   pre-commit run --all-files
+# - Register git hooks: pre-commit install --install-hooks
+
+ci:
```

Comment:
This tells the bot to NOT fix pr's for the time being. it also has it skip black in it's checks because i haven't run black on the repo yet. 

Also we can remove the other github action with code format checks once we are happy with this setup (which is more streamlined!)